django/db/backends/postgresql/operations.py

```diff
@@ -2,6 +2,7 @@
 from functools import lru_cache, partial
 
 from django.conf import settings
+from django.contrib.postgres.fields.serial import SerialFieldMixin
```

Comment:
Ideally, contrib apps would not be imported by core Django, nor require any knowledge within the core. They should act like third-party packages, with no privileged treatment...